src/docs/files/devguide05.md

```diff
@@ -20,54 +22,68 @@
+
+#### Bidding in an Auction
+
+After you have decided the auction you want to participate in, and click on it, you are redirected to the screen below (Keep in mind that the data shown in the below picture will vary for each auction, as all the auctions taking place are independent of each other).
+
+The UI can be dissected in three parts:
+
+- On the top section, you have the main parameters of the auction
+- On the right-hand side, you have data about the ongoing auction, as well as the price graph.
+- On the left-hand side, you can see the parameters you need to fill in to place an order in the auction.
 
 <img src="/assets/GA6.png" alt="drawing" width="500"/>
 
-#### Bidding in an Auction
+But, let's go into details about what each section of the UI means. 
 
-7. After you have decided the auction you want to participate in, and click on it, you are redirected to the screen below (Keep in mind that the data shown in the below picture will vary for each auction, as all the auctions taking place are independent of each other). The UI can be dissected in two parts, the left side and the right side:
+1) This parameter indicates the total amount of tokens the auctioneer is auctioning. This number represents the total amount of tokens available to be bought by bidders.
 
-- On the left side you have data about the ongoing auction, as well as the price graph.
-- On the right hand side you can see the parameters you need to fill in to place an order in the auction.
+2) This parameter represents the token that the auctioneer has selected to be the one accepted for bidding in the auction. This is the asset you need to bid with. 
 
-<img src="/assets/GA7.png" alt="drawing" width="500"/>
+3) This parameter shows a countdown of the duration of the auction. This parameter marks when the auction will end.
 
-8. In the amount section you would need to input the amount of bidding tokens you are willing to bid in the auction. You can either select the amount you want to bid or directly use the max button to bid all the tokens you have.
+4) This parameter shows the closing price of the auction if no more bids are submitted or canceled. As a bidder, you simply need to bid above this price for higher possibility to place a successful bid. 
 
-<img src="/assets/GA8.png" alt="drawing" width="500"/>
+5) This parameter is the Starting price of token auctioned. The auctioneer has decided its value and it establishes the fact that bidders can’t bid under this price.
 
-9. In the price per bid-token/auction token, the dapp sets by default the price that the auctioneers have set to be the minimum price. From that starting price, you can decide what is the price per bid-token/auction token you are willing to offer.
+6) The amount parameter, is the total amount of bidding tokens you are willing to pay for the auctioneer tokens.
```

Comment:
nit: auctioneer's tokens

Reply:
the word total is also not needed, I guess

Reply:
Modified